projects/upb/build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -eu
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+FUZZER_DICTIONARIES="\
+"
+
+NO_VPTR="--copt=-fno-sanitize=vptr --linkopt=-fno-sanitize=vptr"
+EXTRA_BAZEL_FLAGS="--strip=never  $(for f in $CXXFLAGS; do if [ $f != "-stdlib=libc++" ] ; then echo --copt=$f --linkopt=$f; fi; done)"
```

Comment:
This isn't linking against $LIB_FUZZING_ENGINE anywhere so the fuzz target is not a fuzz target at all but is an instrumented binary using whatever some `main` function provided by upb.